src/tools/tests/storage-key-recipe-resolver-test.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {Manifest} from '../../runtime/manifest.js';
+import {assert} from '../../platform/chai-node.js';
+import {StorageKeyRecipeResolver} from '../storage-key-recipe-resolver.js';
+
+describe('recipe2plan', () => {
+  it('Long + Long: If ReadingRecipe is long running, it is a valid use case', async () => {
```

Comment:
My bullet points were intended for a doc rather than unit test names. A more understandable name would be: "Resolves mapping a handle from a long running arc into another long running arc". WDYT?